src/xdocs/config_system_properties.xml

```diff
@@ -33,6 +33,76 @@
         is <a href="property_types.html#boolean">boolean</a> and defaults
         to <code>false</code>.
       </p>
+      <p>
+        Example of usage by xml include to let keep common part of configs in single file:
+        <source>
+$ cat Test.java
+class Test {
+  int i = 0;
+}
+
+$ cat checkstyle-common.xml
+&lt;-- common part of configs --&gt;
+&lt;module name=&quot;FileLength&quot;&gt;
+  &lt;property name=&quot;max&quot; value=&quot;1&quot;/&gt;
+&lt;/module&gt;
+
+$ cat checkstyle.xml
+&lt;-- config for main code --&gt;
+&lt;?xml version=&quot;1.0&quot;?&gt;
+&lt;!DOCTYPE module PUBLIC
+          &quot;-//Checkstyle//DTD Checkstyle Configuration 1.3//EN&quot;
+          &quot;https://checkstyle.org/dtds/configuration_1_3.dtd&quot; [
+    &lt;!ENTITY common SYSTEM &quot;checkstyle-common.xml&quot;&gt;
+]&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+
+    &amp;common;
+
+    &lt;module name=&quot;TreeWalker&quot;&gt;
+        &lt;module name=&quot;MemberName&quot;&gt;
+            &lt;property name=&quot;format&quot; value=&quot;^[a-z][a-zA-Z]+$&quot;/&gt;
+        &lt;/module&gt;
+    &lt;/module&gt;
+&lt;/module&gt;
+
+$ cat checkstyle-test.xml
+&lt;-- config for test code --&gt;
+&lt;?xml version=&quot;1.0&quot;?&gt;
+&lt;!DOCTYPE module PUBLIC
+          &quot;-//Checkstyle//DTD Checkstyle Configuration 1.3//EN&quot;
+          &quot;https://checkstyle.org/dtds/configuration_1_3.dtd&quot; [
+    &lt;!ENTITY common SYSTEM &quot;checkstyle-common.xml&quot;&gt;
+]&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+
+    &amp;common;
+
+    &lt;module name=&quot;TreeWalker&quot;&gt;
+        &lt;module name=&quot;MemberName&quot;&gt;
+            &lt;property name=&quot;format&quot; value=&quot;_[a-z]&quot;/&gt;
+        &lt;/module&gt;
+    &lt;/module&gt;
+
+&lt;/module&gt;
+
+$ java -Dcheckstyle.enableExternalDtdLoad=true -classpath checkstyle-XX.X-all.jar \
+        com.puppycrawl.tools.checkstyle.Main         -c checkstyle.xml Test.java
```

Comment:
I was confused at first why we have the same java command twice with the same output, and then I noticed the config files are different and the membername message was also slightly different.

I think this would make more sense if we split this into 3 blocks and explain each one. First to show how to define configs, 2nd to run with main config, and 3rd to run with test config.

Reply:
I think it would also be good to modify the message and/or id property in each module to mention where it lives, i.e. MemberNameExternal or something like that.

Reply:
do some separation and wording.

I did not make ID and special message to over complicate config and let user focus on what matter most.